.devcontainer/devcontainer.json

```diff
@@ -3,10 +3,11 @@
     "build": {
         "dockerfile": "../Dockerfile",
         "context": "..",
-        "target": "visualizer",
+        "target": "dever",
         "cacheFrom": "ghcr.io/ros-planning/navigation2:main"
     },
     "runArgs": [
+        "--name=nav2"
```

Comment:
makes the process of rebuilding and accessing the container quicker if we don't have to always check the name of the container

Reply:
That fine. Just so long as we don't assume the container name anywhere else.
One shouldn't grow accustomed to hardcoding names of an ephemeral containers.